src/views/developers/developers.jsx

```diff
@@ -92,15 +92,9 @@ var Developers = React.createClass({
                         </FlexRow>
                         <FlexRow className="sidebar-row">
                             <div className="body-copy column">
-                                <h3>ScratchJr</h3>
+                                <h3><FormattedMessage id='developers.jrTitle' /></h3>
```

Comment:
Actually, we ask translators not to translate 'Scratch' or 'ScratchJr' since they're names. Since there's no text other than ScratchJr this one doesn't need translation.